src/bitrot.py

```diff
@@ -180,12 +179,13 @@ def run(self):
         errors = []
         current_size = 0
         missing_paths = self.select_all_paths(cur)
-        paths, total_size = list_existing_paths(
+        hashes_dict_db = self.select_all_hashes(cur)
+        paths_fs, total_size = list_existing_paths(
             b'.', expected=missing_paths, ignored={bitrot_db, bitrot_sha512},
             follow_links=self.follow_links,
         )
 
-        for p in paths:
+        for p in paths_fs:
```

Comment:
It would be nice to wrap this in `sorted()` for reproducible results if it doesn't affect the execution speed too much.